Assets/Scripts/Content/QuestData.cs

```diff
@@ -1179,8 +1179,8 @@ override public string ToString()
     public class Puzzle : Event
     {
         new public static string type = "Puzzle";
-        public string puzzleClass = "slide";
-        public string skill = "{observation}";
+        public StringKey puzzleClass = new StringKey("val", "PUZZLE_SLIDE_CLASS");
```

Comment:
The data on disk and internally should remain the same, only displayed data should change.  If we want to change the way it is stored in the ini we need to at least support back compat with existing quests.  If we are going to to this perhaps moving to an int or PuzzleSlide or something else.  I think translation needs to be involved here.